Makefile

```
@@ -211,6 +211,34 @@ local-dev-up: local-dev-clean node_modules | build-image-quay
 	while ! test -e ./static/build/main-quay-frontend.bundle.js; do sleep 2; done
 	@echo "You can now access the frontend at http://localhost:8080"
 
+QUAY_BUILDER_IMAGE ?= quay.io/projectquay/quay-builder:3.17-unstable
+
+.PHONY: local-dev-extract-builder
+local-dev-extract-builder:
```

checkmake (0.3.2)

[warning] Target body for "local-dev-extract-builder" exceeds allowed length of 5 lines (9).

(maxbodylength)